### DISCUSSION TITLE
Transport endpoint is not connected

### DISCUSSION THREAD

**Discussion by amila-ceranic** [sebastian-hofstaetter/teaching#144] [Advanced IR: Exercise Q&A]
Did anyone have problem with running Part 2 in Colab? After running for 4 hours, the following error occurred: 

"OSError [Errno 107] Transport endpoint is not connected"

**Comment by dblasko** [comment]
Hi,

Does the script you are running perhaps create/write a lot of files/folder in your Google Drive folder? This is not the only reason for such an error, but it happens when there are too many elements in the folder you are manipulating (cf. [https://research.google.com/colaboratory/faq.html#drive-timeout](https://research.google.com/colaboratory/faq.html#drive-timeout) ).
